ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,200 @@
+      of program-approved demonstration projects.
+    </p>
+    <p>
+      This is an agreement between Vanderbilt University Medical Center and authorized demonstration users of data
+      from the <AoUTitle/>.
+    </p>
+    <p>
+      An <strong>authorized demonstration user</strong> is a person who is authorized to access
+      and/or work with registered or
+      controlled tier data from the <AoUTitle/> for the exclusive purpose of a program-approved
+      demonstration project. Authorized demonstration users are limited to trainees, faculty or staff at <AoUTitle/>
+      consortium partner institutions.
+    </p>
+    <p>
+      <strong>Before</strong> they access and/or work with <AoUTitle/> data, authorized demonstration users must:
+      <IndentedOrderedList>
+        <li>complete the <AoUTitle/> research ethics training; and</li>
+        <li>read and attest to this data use agreement</li>
+      </IndentedOrderedList>
+    </p>
+    <p>Please read this agreement carefully and completely before signing.</p>
+    <SecondHeader>As “Authorized Demonstration User” of the <AoUTitle/> data, I
+      will:</SecondHeader>
+    <IndentedUnorderedList>
+      <li>read and adhere to the <AoUTitle/> <a target='_blank' href={CORE_VALUES_URL}>core values</a>.
+      </li>
+      <li>know and follow all applicable US federal, state, and local
+        (applicable in the area where I am conducting research) laws
+        regarding research involving human data and data privacy.
+      </li>
+      <li>respect the privacy of research participants at all times.
+        <IndentedUnorderedList>
+          <IndentedListItem style={{margin: '0.5rem 0'}}>
+            I will <strong>NOT</strong> use or disclose any
+            information that directly identifies one or more participants.
+            <br/>
+            If I become aware of any information that directly identifies one or more
+            participants, I will notify the <AoUTitle/> immediately
+            using the automatic notification system.
+          </IndentedListItem>
+          <IndentedListItem style={{margin: '0.5rem 0'}}>
+            I <strong>will <strong>NOT</strong> attempt</strong> to re-identify research
+            participants or their relatives.
+            <br/>
+            If I unintentionally re-identify participants through the process of my work, I will
+            contact the <AoUTitle/> immediately using the automatic
+            notification system.
+          </IndentedListItem>
+          <IndentedListItem style={{margin: '0.5rem 0'}}>
+            If I become aware of any uses or disclosures of <AoUTitle/> data that
+            could endanger the security or privacy of research participants, I will contact the
+            <AoUTitle/> immediately using the appropriate process.
+          </IndentedListItem>
+          <IndentedListItem>provide a meaningful description of my research purpose when establishing my <AoUTitle/>
```

Comment:
Breakline before `my`, currently this shows up as one word.

Also, this should be de-indented one more (looks like it is single indent rather than double indent in the doc)